src/directive.ts

```diff
@@ -194,9 +202,9 @@ function legacyPartToPart(part: legacyLit.Part): Part {
     return new EventPart(part);
   } else if (part instanceof legacyLit.BooleanAttributePart) {
     return new BooleanAttributePart(part);
-  } else if (
-      part instanceof legacyLit.PropertyPart ||
-      part instanceof legacyLit.AttributePart) {
+  } else if (part instanceof legacyLit.PropertyPart) {
+    return new PropertyPart(part);
```

Comment:
Was this a bug that we never created a PropertyPart? Do we have test coverage here?

Reply:
I'm not sure if there was / wasn't a reason it wasn't originally included. I can't find tests for this file.

Reply:
I've added a basic set of tests that check that `.type` of the `PartInfo` given to the directive's constructor is the expected value for each type of part. For `PropertyPart` specifically, there's nothing else to test.